playbook/app/pb_kits/playbook/utilities/globalProps.ts

```diff
@@ -62,6 +62,7 @@ type FlexWrap = {
 
 type Hover = Shadow & {
   background?: string,
+  color?: string,
```

Comment:
Should we add this addition to the Hover global prop to the Hover docs on the visual guidelines page? That can definitely be a follow-up story though, doesn't have to be on this PR

cc: @jasperfurniss